http/mdm/mdm.go

```diff
@@ -37,12 +37,21 @@ func CheckinHandler(svc service.Checkin, logger log.Logger) http.HandlerFunc {
 		}
 		respBytes, err := service.CheckinRequest(svc, mdmReqFromHTTPReq(r), bodyBytes)
 		if err != nil {
-			logger.Info("msg", "check-in request", "err", err)
+			logs := []interface{}{"msg", "check-in request"}
 			httpStatus := http.StatusInternalServerError
 			var statusErr *service.HTTPStatusError
 			if errors.As(err, &statusErr) {
 				httpStatus = statusErr.Status
+				err = statusErr.Unwrap()
 			}
+			// manualy unwrapping the `StatusErr` is not necessary as `errors.As` manually unwraps
+			var parseErr *mdm.ParseError
+			if errors.As(err, &parseErr) {
+				logs = append(logs, "content", string(parseErr.Content))
+				err = statusErr.Unwrap()
```

Comment:
just a lil bit more context :)

```suggestion
				err = fmt.Errorf("parse error: %w", parseErr.Unwrap())
```